src/StartingEnv.hs

```diff
@@ -220,6 +220,7 @@ dynamicModule = Env { envBindings = bindings
                     , addCommand (SymPath path "load") 1 commandLoad "loads a file into the current environment." "(load \"myfile.carp\")"
                     , addCommand (SymPath path "load-once") 1 commandLoadOnce "loads a file and prevents it from being reloaded (see `reload`)." "(load-once \"myfile.carp\")"
                     , addCommand (SymPath path "expand") 1 commandExpand "expands a macro and prints the result." "(expand '(when true 1)) ; => (if true 1 ())"
+                    , addCommand (SymPath path "arch") 0 commandArch "prints the native architecture (as returned by the Haskell function `System.Info.arch`)." "(arch)"
```

Comment:
I think `target` is missing here?

Reply:
Missing from where?

Reply:
It‘s defined as a primitive, but not wired up in the registration here, unless I‘m missing something?

Reply:
Oh, right. Fixed.